import-automation/executor/app/executor/import_executor.py

```diff
@@ -351,6 +351,78 @@ def _get_latest_version(self, import_dir: str) -> str:
         latest_version = blob.download_as_text()
         return f'gs://{bucket.name}/{import_dir}/{latest_version}'
 
+    def _get_import_input_files(self, import_input, absolute_import_dir):
+        input_files = []
+        template_mcf = []
+        if pattern := import_input.get('template_mcf'):
+            template_mcf = glob.glob(os.path.join(absolute_import_dir, pattern))
+            import_prefix = os.path.splitext(os.path.basename(
```

Comment:
declare import_prefix outside the condition as well o/w line 372 will raise an exception.
`import_prefix = ''`